sympy/printing/tests/test_ccode.py

```diff
@@ -298,14 +297,12 @@ def test_ccode_Indexed():
     A = IndexedBase('A')[i, j]
     B = IndexedBase('B')[i, j, k]
 
-    with warns_deprecated_sympy():
-        p = CCodePrinter()
-    p._not_c = set()
+    p = C99CodePrinter()
 
     assert p._print_Indexed(x) == 'x[j]'
     assert p._print_Indexed(A) == 'A[%s]' % (m*i+j)
     assert p._print_Indexed(B) == 'B[%s]' % (i*o*m+j*o+k)
-    assert p._not_c == set()
+    # assert p._not_c == set()
```

Comment:
Should this line just be removed?

Reply:
I am not sure why this was present previously. If it can be removed without any issue i'll add a commit for the same

Reply:
It was added here:
https://github.com/sympy/sympy/pull/5

I don't see what purpose it serves.

Reply:
So should I remove it and make a PR?

Reply:
This PR already removes it. You've just left it in commented out when perhaps it should just be deleted.

I'm not sure what purpose it originally serves though. I see it is also used in some other modules:
```console
$ git grep _not_c
sympy/core/expr.py:        # numbers (see test_is_not_constant test), giving False when it
sympy/core/tests/test_arit.py:def test_Mul_does_not_cancel_infinities():
sympy/printing/tests/test_ccode.py:    p._not_c = set()
sympy/printing/tests/test_ccode.py:    assert p._not_c == set()
sympy/printing/tests/test_glsl.py:    p._not_c = set()
sympy/printing/tests/test_glsl.py:    assert p._not_c == set()
sympy/printing/tests/test_jscode.py:    p._not_c = set()
sympy/printing/tests/test_jscode.py:    assert p._not_c == set()
```

Reply:
I have deleted the commented line